src/components/main/Main.js

```diff
@@ -17,12 +17,18 @@ function Main(props) {
       }
     }
 
+    if (selectedTypeIds.length > 0) {
+      for(let i=0; i < selectedTypeIds.length; i++) {
+        url += `&type_id=${selectedTypeIds[i]}`
+      }
+    }
```

Comment:
xịn hơn có thể dùng for in hoặc for of cho ngắn https://anonystick.com/blog-developer/su-khac-biet-giua-forin-forof-and-foreach-trong-javascript-2020041337746860